src/turnkeyml/analyze/script.py

```diff
@@ -257,13 +260,26 @@ def explore_invocation(
 
         return
 
-    # Initialize build and benchmark status to "not started"
-    stats.save_model_eval_stat(
-        fs.Keys.BUILD_STATUS, build.FunctionStatus.NOT_STARTED.value
-    )
-    stats.save_model_eval_stat(
-        fs.Keys.BENCHMARK_STATUS, build.FunctionStatus.NOT_STARTED.value
-    )
+    # Initialize build and benchmark status to "not started" or
+    # "not applicable" depending on whether that action is part of the
+    # evaluation
+    if runtime_info["build_required"]:
+        stats.save_model_eval_stat(
+            fs.Keys.BUILD_STATUS, build.FunctionStatus.NOT_STARTED.value
+        )
+    else:
+        stats.save_model_eval_stat(
+            fs.Keys.BUILD_STATUS, build.FunctionStatus.NOT_APPLICABLE.value
```

Comment:
I personally think that adding another status here actually makes things more confusing than it clarifies things. 

Here are some options ordered according to my personal preference:

### Option 1:  Just keep `not started` and don't add `not_applicable`
My understanding is that it is clear that if something is `not_applicable` it will not start. Therefore, simply using `not_started` instead would be clear.

### Option 2: Keep things the way they are
Not super clear, but verbosity is low.

### Option 3: Rename `not_started` to `applicable_but_not_started`
Way too verbose, but at least it is clear.

Please feel free to choose the option you prefer. No strong feelings here.

Reply:
Implemented Option 4 in https://github.com/onnx/turnkeyml/pull/106/commits/76fb2683d8a31a9c1f6f4b050c717d80ee2192a3

Build and benchmark status will simply be omitted if those actions are not part of the evaluation:

```
evaluations:
  x86_ort:
    build_status: successful
    device_type: x86
    iterations: 100
    runtime: ort
  x86_torch-eager:
    benchmark_status: successful
    build_name: linear_selftest_76af2f62
    device: AMD Ryzen 9 7940HS w/ Radeon 780M Graphics
    device_type: x86
    iterations: 100
    latency_units: milliseconds (ms)
    mean_latency: 0.013790999999985232
    runtime: torch-eager
    runtime_version: 2.1.2+cpu
    throughput: 72511.05793641294
    throughput_units: inferences per second (IPS)
```